### PR TITLE
project list page without db connection

### PR DESCRIPTION
**Pull request by dyeoman2**

### CODE REVIEW

File: pages/projects.js [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+EnhancedTableHead.propTypes = {
+  numSelected: PropTypes.number.isRequired,
+  onRequestSort: PropTypes.func.isRequired,
+  onSelectAllClick: PropTypes.func.isRequired,
+  order: PropTypes.oneOf(['asc', 'desc']).isRequired,
+  orderBy: PropTypes.string.isRequired,
+  rowCount: PropTypes.number.isRequired,
+};
+
+const EnhancedTableToolbar = (props) => {
```
Comment (theTschida) [inline pages/projects.js:182]: I think we should make this into a separate component to modularize the code and promote maintainability.

File: pages/projects.js [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+      </Button>
+    </Toolbar>
+  );
+};
+
+EnhancedTableToolbar.propTypes = {
+  numSelected: PropTypes.number.isRequired,
+};
+
+export default function EnhancedTable() {
```
Comment (theTschida) [inline pages/projects.js:226]: I think we should make this into a separate component to modularize the code and promote maintainability.

File: pages/projects.js [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+  },
+  {
+    id: 'createdAt',
+    numeric: false,
+    disablePadding: false,
+    label: 'Created At',
+  },
+];
+
+function EnhancedTableHead(props) {
```
Comment (theTschida) [inline pages/projects.js:132]: I think we should make this into a separate component to modularize the code and promote maintainability.

File: pages/projects.js [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+    id,
+    name,
+    shared,
+    teamMembers,
+    updatedAt,
+    createdAt,
+  };
+}
+
+const rows = [
```
Comment (theTschida) [inline pages/projects.js:52]: If this is mock data, I think we should comment that, so it is clear during refactor.

File: pages/projects.js [hunk 5]
```diff
@@ -0,0 +1,357 @@
+import * as React from 'react';
+import { useRouter } from 'next/router';
+
+import PropTypes from 'prop-types';
```
Comment (theTschida) [inline pages/projects.js:4]: Would it be cleaner to destructure items in one import from material-ui/core? Is there a downside to the approach of destructuring vs one line importing?

**Review by theTschida — CHANGES_REQUESTED** [review]
I think it would be cleaner and more maintainable if this large file were broken into components. If we reserve our pages for fully realized components and keep them clear of functional code, they should be more easily augmented in the the future.

File: pages/projects.js (hunk 5 above)
Comment (dyeoman2) [reply pages/projects.js:4]: done

File: pages/projects.js (hunk 4 above)
Comment (dyeoman2) [reply pages/projects.js:52]: done

File: pages/projects.js (hunk 1 above)
Comment (dyeoman2) [reply pages/projects.js]: done

File: pages/projects.js (hunk 3 above)
Comment (dyeoman2) [reply pages/projects.js]: done

File: pages/projects.js (hunk 2 above)
Comment (dyeoman2) [reply pages/projects.js:78]: done

**Review by theTschida — CHANGES_REQUESTED** [review]
Please move the Table folder out of utils, and into the main components folder. Otherwise, it looks great!

**Pull request merged**